pallets/dao/src/lib.rs

```diff
@@ -219,22 +236,22 @@ pub mod pallet {
 	#[pallet::storage]
 	#[pallet::getter(fn member_of)]
 	#[pallet::unbounded]
-	/// Storage item that indicates which DAO's a user belongs to [AccountID, Vec]
-	pub(super) type MemberOf<T: Config> = StorageMap<_, Twox64Concat, T::AccountId, Vec<OrganizationIdOf<T>>, ValueQuery>;
+	/// Storage item that indicates which DAO's a user belongs to [AccountID, BoundedVec<OrganisationId>]
+	pub(super) type MemberOf<T: Config> = StorageMap<_, Twox64Concat, T::AccountId, BoundedOrgPerMember<T>, ValueQuery>;
 
 	#[pallet::storage]
 	#[pallet::getter(fn applicants_to_organization)]
 	#[pallet::unbounded]
-	/// Storage Map to indicate which user agree with a proposed Vision of an Organisation [DaoId, Vec[Account]]
-	pub(super) type ApplicantsToOrganization<T: Config> = StorageMap<_, Twox64Concat, OrganizationIdOf<T>, Vec<T::AccountId>, ValueQuery>;
+	/// Storage Map to indicate which user agree with a proposed Vision of an Organisation [OrganizationId, BoundedVec[Account]]
+	pub(super) type ApplicantsToOrganization<T: Config> = StorageMap<_, Twox64Concat, OrganizationIdOf<T>, BoundedApplicantsPerOrg<T>, ValueQuery>;
 
 	#[pallet::event]
 	#[pallet::generate_deposit(pub(super) fn deposit_event)]
 	pub enum Event<T: Config> {
-		/// Vision signed [AccountID, DaoIdOf]
+		/// Vision signed [AccountID, OrganizationIdOf]
```

Comment:
Why we need the Of? Why not `OrganizationId`

Reply:
https://github.com/UniversalDot/universal-dot-node/pull/96/commits/f81cfcd7f65c9ad8c93a8876011ca6fd2c0252b7